README.md

```diff
@@ -1,2 +1,18 @@
-# powerflows-dmn-model
+# About Power Flows DMN Model
 Power Flows DMN Model
+
+# How to contribute to the repository
+Contributors wanted to join Power Flows project have to comply with a few rules: 
+
+* A preferred IDE is IntelliJ IDEA with default formatting styles
+* Committed source code must be high quality
+* Committed source code must be formatted
+* A commit has to solve an issue from the product backlog (if missing the issue there, please add previously)
```

Comment:
' if the issue is missing, add one before creating pr. Refer the issue in commit summary.'

Reply:
Improved